Additional metrics (#920)
imgproxy/imgproxy: docs/new_relic.md

* CPU and memory usage
* Response time
* Queue time
* Image downloading time
* Image processing time
* Errors that occurred while downloading and processing an image

Additionally, imgproxy sends the following metrics over [Metrics API](https://docs.newrelic.com/docs/data-apis/ingest-apis/metric-api/introduction-metric-api/):

> * `imgproxy.requests_in_progress`: a number of requests currently being in progress

Reviewer: ```suggestion
* `imgproxy.requests_in_progress`: the number of requests currently in progress
```

> * `imgproxy.images_in_progress`: a number of images currently being in progress

Reviewer: ```suggestion
* `imgproxy.images_in_progress`: the number of images currently in progress
```
